src/port1.0/portbuild.tcl

```diff
@@ -91,8 +91,9 @@ proc portbuild::build_getmaketype {args} {
     global xcodebuildcmd
     if {[option build.type] eq "default"} {
         return [findBinary make $portutil::autoconf::make_path]
-    }]
-    if {[option use_xcode] eq "yes" && $xcodebuildcmd eq "none"} {
+    }
+    # TODO: Implement reliable way of checking for Xcode
+    if {([option build.type] eq "xcode" || [option use_xcode] eq "yes") && $xcodebuildcmd eq "none"} {
```

Comment:
Didn't you replace this with changing the default value of `use_xcode` if `build.type` is `xcode`? If so, shouldn't this commit be dropped from the history of the PR?